packages/server/src/queue/queue.entity.ts

```diff
@@ -82,20 +79,10 @@ export class QueueModel extends BaseEntity {
     return open;
   }
 
-  @Expose()
-  get queueSize(): number {
-    if (!this.questions) {
-      // if you're getting this, make sure you're loading `questions` in relations when you're getting a queue
-      // or you're adding questions to your QueueModel.create as []
-      throw new Error(
-        "Questions weren't loaded when trying to grab queue size",
-      );
-    }
-    return this.questions?.filter(
-      (q) =>
-        q.status === OpenQuestionStatus.Drafting ||
-        q.status === OpenQuestionStatus.Queued,
-    ).length;
+  queueSize: number;
+
+  async addQueueSize(): Promise<void> {
+    this.queueSize = await QuestionModel.openInQueue(this.id).getCount();
```

Comment:
The way this is written, `queueSize` now includes all open questions, even `TADeleted` etc. I think we should just revamp all this in the upcoming Priority Queue ticket